exercises/es/solution_02_13.py

```diff
@@ -3,25 +3,20 @@
 
 nlp = spacy.load("es_core_news_sm")
 doc = nlp(
-    "Cuando Pac-Man debutó en Tokio, en 1980, nadie podría haber predicho "
+    "Cuando pac-man debutó en Tokio, en 1980, nadie podría haber predicho "
     "que se convertiría en el videojuego más exitoso de todos los tiempos. "
     "Hoy, 40 años después, aun sigue sorprendiendo. Su desarrolladora, "
     "Bandai Namco, ha anunciado novedades en el marco del aniversario del "
-    "juego. La celebración del 40 aniversario de Pac-man en 2020 incluirá "
+    "juego. La celebración del 40 aniversario de pac-Man en 2020 incluirá "
     "el debut de una nueva canción temática, compuesta por el famoso artista "
     "japonés de Techno Ken Ishii. Además de estas novedades, Bandai Namco "
-    "publicará nuevas versiones del videojuego. La primera será Pac-Man Live "
+    "publicará nuevas versiones del videojuego. La primera será pac-man Live "
     "Studio, en Twitch, en colaboración con Amazon Games."
 )
 
 # Crea los patrones
-pattern1 = [{"LOWER": "bandai"}, {"IS_TITLE": True, "POS": "PROPN"}]
-pattern2 = [
-    {"LOWER": "pac"},
-    {"TEXT": "-"},
-    {"LOWER": "man"},
-    {"POS": "PROPN", "OP": "*"},
-]
+pattern1 = [{"LIKE_NUM": True}, {"POS": "NOUN"}]
+pattern2 = [{"LOWER": "pac-man"}, {"IS_TITLE": True, "OP": "*"}]
```

Comment:
Same here, I think the example could be slightly simpler and not ask for `{"IS_TITLE": True, "OP": "*"}` explicitly.